index.js

```diff
@@ -1,9 +1,13 @@
-var instructions = require('./instructions.json');
+module.exports = function(_language, _version) {
+    var version = _version || 'v5';
+    var language = _language || 'en';
 
-if (Object !== instructions.constructor) throw 'instructions must be object';
+    var instructions = require('./instructions').get(language);
+    if (Object !== instructions.constructor) throw 'instructions must be object';
 
-module.exports = function(_version) {
-    var version = _version || 'v5';
+    function capitalizeFirstLetter(string) {
+        return string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
If it's depending on the context, I agree that captilizing should be handled by client code. But if we do that, shouldn't all translated strings be lowercase?